src/components/App/components/IdleLogout.vue

```diff
@@ -62,11 +64,13 @@ export default {
   methods: {
     initTimer () {
       const warnInterval = this.$store.getters.forceLogoutDelay
-      this.warnEndTimestamp = moment().add(warnInterval, 'seconds').unix()
+      this.warnEndTimestamp = moment()
+        .add(warnInterval, globalize('idle-logout.seconds'))
+        .unix()
       this.countdownInterval = setInterval(() => {
         const msLeft = moment
-          .duration(this.warnEndTimestamp - moment().unix(), 'seconds')
-          .asMilliseconds()
+          .duration(this.warnEndTimestamp - moment().unix(),
```

Comment:
Why did you decide to localize this? Do you know how to work 'duration' method?

Reply:
You are not careful again

Reply:
done